src/Card/Card.types.tsx

```diff
@@ -0,0 +1,106 @@
+// Card.component
```

Comment:
I do like the types living in their own file. Files should be called either `index.d.ts` or `Cards.d.ts`. Thoughts ? @martindidiego @fhern077

Reply:
Don’t mind either name as long as we’re consistent. `index.d.ts` would be following convention, though, so I prefer that one if I had to choose.

Reply:
I think putting in Cards.d.ts would be best and then barrel all types to the index.d.ts that way it makes it easier to find relevant types + we can comply with convention.

Reply:
Yup this should be called `Cards.d.ts`. It will interpret it as the declaration file instead of `Cards.types.tsx` @AndresMurguido001

Reply:
Styled system changed how types are declared.